packages/snap/tests/rpc/onRpcRequest.spec.ts

```diff
@@ -1624,4 +1618,180 @@ describe('onRpcRequest', () => {
       expect.assertions(1);
     });
   });
+
+  describe('createVC', () => {
+    it('should succeed creating a VC', async () => {
+      const vc = (await onRpcRequest({
+        origin: 'localhost',
+        request: {
+          id: 'test-id',
+          jsonrpc: '2.0',
+          method: 'createVC',
+          params: {
+            minimalUnsignedCredential: unsignedMinimalCredential,
```

Comment:
Can we name this the same way ?

Reply:
Also we have this already in  the `exampleTestVCPayload` variable. Also the `exampleTestVCPayload` is missing the `@context` field.